lesson14/script.js

```diff
@@ -0,0 +1,95 @@
+const showLoadingImg = () => {
+    const div        = document.createElement('div');
```

Comment:
It is better to have specific names for variables. For example, "loadingPlace".

Reply:
Thank you for your comment!
I renamed  the variables.
https://github.com/nakagawamai/js-lesson/pull/16/commits/c9329c8c192860a0856fb4ccb32bbca28c07c96b